modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/index.tsx

```diff
@@ -106,7 +106,7 @@ const CustomView: React.FC<React.HTMLAttributes<HTMLElement>> = () => {
 			const {
 				items: objectFields,
 			}: {
-				items: TObjectField[];
+				items: ObjectField[];
 			} = (await objectFieldsResponse.json()) as any;
```

Comment:
The same here. Avoid using any type. Sorry for that rsrs Milton would ask for this :sweat_smile: